### PR TITLE
[tidb-cdc] TiDB CDC dealUnsignedColumnValue throw NullPointerException.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Fix issue https://github.com/ververica/flink-cdc-connectors/issues/1442 .

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@ruanhang1993 Pls cc . If you are avalible.

**Comment by GOODBOY008** [comment]
@Jiabao-Sun Pls cc.Thank you!

**Review by Jiabao-Sun — COMMENTED** [review]
Thanks @GOODBOY008 for this contribution.
I left a few comments.

File: flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/table/RowDataTiKVEventDeserializationSchemaBase.java [hunk 1]
```diff
@@ -527,10 +527,6 @@ public Object convert(
                     } else {
                         int offset = columnInfo.getOffset();
                         org.tikv.common.types.DataType type = columnInfo.getType();
-                        if (type.isUnsigned()) {
-                            ((Object[]) object)[offset] =
-                                    dealUnsignedColumnValue(type, ((Object[]) object)[offset]);
-                        }
```
[the remark this answers is not in the document]
Comment (GOODBOY008) [reply flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/table/RowDataTiKVEventDeserializationSchemaBase.java:533]: I think add this logic to `wrapIntoNullableConverter` will be better and easy to understand.

[the remark this answers is not in the document]
Comment (Jiabao-Sun) [reply flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/table/RowDataTiKVEventDeserializationSchemaBase.java:533]: From my point of view this logic is more like preprocessing than for handling non-null values.

[the remark this answers is not in the document]
Comment (GOODBOY008) [reply flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/table/RowDataTiKVEventDeserializationSchemaBase.java:533]: How about move this logic to method `convertField` ?

[the remark this answers is not in the document]
Comment (Jiabao-Sun) [reply flink-connector-tidb-cdc/src/main/java/com/ververica/cdc/connectors/tidb/table/RowDataTiKVEventDeserializationSchemaBase.java:533]: > How about move this logic to method `convertField` ?

That's good.

**Review by Jiabao-Sun — APPROVED** [review]
Thanks @GOODBOY008, LGTM.

**Pull request merged**